src/test/ml-modules/root/test/suites/ClaimSearchSuite/test.sjs

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const test = require("/test/test-helper.xqy");
+
+function runSearch(search, start = 0, limit = 100) {
+  return fn.head(
+    xdmp.invoke('/data-services/claim/search.sjs', {
+      search: xdmp.quote(Array.isArray(search) ? search : [search]),
+      start,
+      limit,
+    }),
+  );
+}
+
+const baseLevelCode = 'http://snomed.info/sct/138875005'; // Base code for all of SNOMED-CT
+const psychCode = 'http://snomed.info/sct/74732009';
+const majorDepressionCode = 'http://snomed.info/sct/36923009';
+
+const baseLevelNarrowerSearch = runSearch({
```

Comment:
Suggest a comment to orient people on above/below like:

// Query for all diagnoses that are more specific than the general clinical finding of 'mental disorder' as classified in the SNOMED taxonomy. 
// This behavior corresponds to the 'below' FHIR search modifier.